### PR TITLE
Add `osu! community meeting: Performance Points and Star Rating Development Q&A` news post

### PR DESCRIPTION
**Pull request by mangomizer**
Please write a better intro and ending for me.
I also forgot to put .md at the end of the file, oops

### CODE REVIEW

File: news/2022-08-24-community-meeting-pp-and-sr-developments [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| 10:00 UTC | osu!mania Q&A panel | [Click me to be notified on Discord!](pending link) |
+| 11:00 UTC | osu!catch Q&A panel | [Click me to be notified on Discord!](pending link) |
+
+---
+
+Join us for a hopefully insightful event! Most importantly, be sure to spread the word to anyone you can - this literally affects everyone.
+
+See you all soon!
+
+—peformance points committee
```
Comment (0x84f) [inline news/2022-08-24-community-meeting-pp-and-sr-developments:43]:
```suggestion
—performance points committee
```
lil thing

**Comment by Ephemeralis** [comment]
This is scheduled for Thursday (2022-08-25) as things currently stand, for the record.

File: news/2022-08-24-community-meeting-pp-and-sr-developments (hunk 1 above)
Comment (mangomizer) [reply news/2022-08-24-community-meeting-pp-and-sr-developments]: Fixed in 6b998bc

**Comment by peppy** [comment]
@Ephemeralis is there a reason this is being delayed? it seems important to get it out in a timely manner so people can make themselves available if they want to participate.

**Comment by Ephemeralis** [comment]
This should be good to go now, assuming the twitch/youtube links are correct (waiting for mango to verify). Definitely needs to go out today at the latest.

File: news/2022-08-26-community-meeting-pp-and-sr-developments.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Got some questions about the future direction of your game mode's Performance Points or Star Rating development? Ask them live during an upcoming Q&A panel hosted on the osu!dev Discord server!
+
+![](https://assets.ppy.sh/media/generic-header.jpg)
+
+Due to popular demand, the [Performance Points Committee](/wiki//People/Performance_Points_Committee) (and associates) have decided to host a series of community meetings to address any questions or concerns you may have about the latest developments surrounding performance points and star rating!
+
+Here's a brief rundown on how the community meetings will work:
+
+- There will be a Q&A panel for each of the osu! game modes, lasting 1 - 1.5 hours each.
```
Comment (Walavouchey) [inline news/2022-08-26-community-meeting-pp-and-sr-developments.md:15]:
```suggestion
- There will be a Q&A panel for each of the osu! game modes, lasting 1–1.5 hours each.
```

**Review by Walavouchey — COMMENTED** [review]
very quick skim but lgtm

**Review by Walavouchey — DISMISSED** [review]

File: news/2022-08-26-community-meeting-pp-and-sr-developments.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Due to popular demand, the [Performance Points Committee](/wiki//People/Performance_Points_Committee) (and associates) have decided to host a series of community meetings to address any questions or concerns you may have about the latest developments surrounding performance points and star rating!
+
+Here's a brief rundown on how the community meetings will work:
+
+- There will be a Q&A panel for each of the osu! game modes, lasting 1–1.5 hours each.
+- These panels will be hosted on the [official osu!dev Discord server](https://discord.gg/ppy) through a Discord stage channel.
+- During each panel, audience members can raise their hand to be invited onto the stage to ask their question. Audience members can also type their questions in the relevant thread, or pre-submit their questions [here](https://forms.gle/Q3YSYZF5ZCEuqXtZ7) if they cannot attend.
+
+Below is the schedule for the community meetings. You can tune in live through the Discord stage channel, and we'll update this newspost with links to the talks should they be recorded or streamed for those who can't make it.
```
Comment (Walavouchey) [inline news/2022-08-26-community-meeting-pp-and-sr-developments.md:19]:
```suggestion
Below is the schedule for the community meetings. You can tune in live through the Discord stage channel, and we'll update this news post with links to the talks should they be recorded or streamed for those who can't make it.
```

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**